test/k8s/datapath_configuration.go

```diff
@@ -171,7 +171,7 @@ var _ = Describe("K8sDatapathConfig", func() {
 		SkipItIf(func() bool {
 			// IPsec + encapsulation requires Linux 4.19.
 			// We also can't disable KPR on GKE at the moment (cf. #16597).
-			return helpers.RunsWithoutKubeProxy() || helpers.DoesNotRunOn419OrLaterKernel() || helpers.RunsOnGKE()
+			return helpers.RunsWithoutKubeProxy() || helpers.DoesNotRunOn419OrLaterKernel() || helpers.RunsOnGKE() || helpers.RunsOnAKS()
```

Comment:
Cannot specify local-router-ipv4 or local-router-ipv6 with enable-ipsec is the error message, so skipping this as well